orange/src/main/java/com/hartwig/hmftools/orange/algo/purple/ChromosomalRearrangementsDeterminer.java

```diff
@@ -0,0 +1,93 @@
+package com.hartwig.hmftools.orange.algo.purple;
+
+import java.util.List;
+
+import com.hartwig.hmftools.common.purple.PurpleCopyNumber;
+import com.hartwig.hmftools.common.genome.refgenome.RefGenomeCoordinates;
+import com.hartwig.hmftools.datamodel.orange.OrangeRefGenomeVersion;
+
+import org.jetbrains.annotations.NotNull;
+
+final class ChromosomalRearrangementsDeterminer
+{
+    private final RefGenomeCoordinates refGenomeCoordinates;
+
+    public ChromosomalRearrangementsDeterminer(@NotNull final OrangeRefGenomeVersion refGenomeVersion)
```

Comment:
Minor but we typically only use constructors that take the exact properties as input.

This function I would make into a public static factory method (e.g. see `RefGenomeVersion.from`)

Reply:
Could you elaborate what you mean here? Trying to understand and tried some things, but I can't get it right

Reply:
I have just pushed the change, please take a look! Easier to explain that way then by commenting I thought.

Reply:
Yes, thanks, this helps to understand it!